components/usage/pkg/stripe/stripe_test.go

```diff
@@ -89,3 +91,11 @@ func TestCustomerQueriesForTeamIds_MultipleQueries(t *testing.T) {
 		})
 	}
 }
+
+func TestStartOfNextMonth(t *testing.T) {
+	nextMonth := time.Now().AddDate(0, 1, 0).Month()
+	actualStartOfNextMonth := time.Time(time.Date(2022, nextMonth, 1, 0, 0, 0, 0, time.UTC))
```

Comment:
This will start to fail in December 2022

Reply:
Good catch! Will use a programmatic year.